.github/workflows/list-versions.yml

```diff
@@ -0,0 +1,56 @@
+name: List running service versions
+run-name: List service versions on ${{ github.event.inputs.environment }}
+
+on:
+  # Only runs manually
+  workflow_dispatch:
+    inputs:
+      environment:
+        description:  Which AWS Account to use
+        type: choice
+        required: true
+        options:
+        - dev
+        - test
+        - uat
+        - prod
+
+jobs:
+  list-versions:
+    permissions:
+      id-token: write # This is required for requesting the JWT
```

Comment:
Do we need `id-token` for this workflow?

Reply:
If it's OK I'll book it in so I can test in main, then check and raise another PR to remove if if not needed.